extmod/modbluetooth.h

```diff
@@ -414,6 +415,9 @@ void mp_bluetooth_gap_on_connected_disconnected(uint8_t event, uint16_t conn_han
 // Call this when any connection parameters have been changed.
 void mp_bluetooth_gap_on_connection_update(uint16_t conn_handle, uint16_t conn_interval, uint16_t conn_latency, uint16_t supervision_timeout, uint16_t status);
 
+// Call this when any connection subscribes or unsubscribes to a characteristic
+void mp_bluetooth_gatts_on_subscription_update(uint16_t conn_handle, uint16_t chr_value_handle, uint16_t cur_notify, uint16_t cur_indicate);
```

Comment:
Just `value_handle` for consistency with the other methods.

No need to distinguish from `prev_notify` in the NimBLE struct so I think just `notify` and `indicate` (and these should be bools).